Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM node:12-alpine
+
+RUN apk update && \
+    apk --no-cache upgrade && \
+    apk add --no-cache git \
+                       openssh-client \
```

Comment:
do you think we need it?

Reply:
It's useful if we work with private repo. 
Probably not as we pushed everything public and this repo is public too.

Reply:
Oh, right. But why do we need python? To build some npm dependency?